docs/conf.py

```diff
@@ -39,9 +39,16 @@
     "gwcs": ("https://gwcs.readthedocs.io/en/latest/", None),
     "astropy": ("https://docs.astropy.org/en/stable/", None),
     "tweakwcs": ("https://tweakwcs.readthedocs.io/en/latest/", None),
+    "imagestats": ("https://stsciimagestats.readthedocs.io/en/latest/", None),
+    "spherical_geometry": ("https://spherical-geometry.readthedocs.io/en/latest/", None),
 }
 
-nitpick_ignore = [("py:class", "optional"), ("py:class", "np.ndarray")]
+nitpick_ignore = [
+    ("py:class", "optional"),
+    ("py:class", "np.ndarray"),
+    ("py:class", "stsci.imagestats.ImageStats"),  # intersphinx isn't working here
+    ("py:class", "spherical_geometry.polygon.SphericalPolygon"), # intersphinx isn't working here
```

Comment:
These should be working with the intersphinx links I added. I believe the fix may require some fixes in the documentation for those packages.

It is possible something is off in the `stcal` docs causing the link failure. Please feel free to push changes to attempt to fix this.